src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.csproj

```diff
@@ -18,7 +18,15 @@
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <IncludeBuildOutput>False</IncludeBuildOutput>
     <IncludeSymbols>True</IncludeSymbols>
-  </PropertyGroup>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
TIL about snupkg being a new "format" (same thing, just some stricter rules), and the only one NuGet will host symbols from. Our AppVeyor build will need to be updated to pick up the extension as an artifact though.